doc-content/enterprise-only/admin-and-config/clean-log-proc.adoc

```diff
@@ -0,0 +1,63 @@
+[id='clean-log-proc']
+= Removing logs from the database
+
+Use `LogCleanupCommand` executor command to clean up the data that is using the database space. The `LogCleanupCommand` consists of logic to automatically clean up all or selected data.
+
+There are several configuration options that you can use with the `LogCleanupCommand`:
+
+[id='logcleanupcommand_parameters_{context}']
+.LogCleanupCommand parameters table
+[cols="1,2,1", options="header"]
+|===
+| Name
+| Description
+| Is Exclusive
+
+|`SkipProcessLog`
+|Indicates whether process and node instances, and process variables log cleanup is skipped when the command runs. The default value is `false`.
+|No, it is used with other parameters.
+
+|`SkipTaskLog`
+|Indicates if the task audit and event log cleanup are skipped. The default value is `false`.
+|No, it is used with other parameters.
+
+|`SkipExecutorLog`
+|Indicates if {PRODUCT} executor entries cleanup is skipped. The default value is `false`.
+|No, it is used with other parameters.
+
+|`SingleRun`
+|Indicates if a job routine runs only once. The default value is `false`.
+|No, it is used with other parameters.
+
+|`NextRun`
+|Schedules the next job execution. The default value is `24h`.
+
+For example, set to `12h` for jobs to be executed every 12 hours. The schedule is ignored if you set *SingleRun* to `true`, unless you set both *SingleRun* and *NextRun*. If both are set, the *NextRun* schedule takes priority. The ISO format can be used to set the precise date.
+|No, it is used with other parameters.
+
+|`OlderThan`
+|Logs that are older than the specified date are removed. The date format is `YYYY-MM-DD`. Usually, this parameter is used for single run jobs.
+|Yes, it is not used with `OlderThanPeriod` parameter.
+
+|`OlderThanPeriod`
+|Logs that are older than the specified timer expression are removed. For example, set `30d` to remove logs, which are older than 30 days.
+|Yes, it is not used with `OlderThan` parameter.
+
+|`ForProcess`
+|Specifies process definition ID for logs that are removed.
+|No, it is used with other parameters.
+
+|`ForDeployment`
+|Specifies deployment ID of the logs that are removed.
+|No, it is used with other parameters.
+
+|`EmfName`
+|Persistence unit name that is used to perform operation deletion.
```

Comment:
I would use .... to perform **delete** operation.

Reply:
Done